src/GoTrueClient.ts

```diff
@@ -1538,7 +1545,11 @@ export default class GoTrueClient {
     let codeChallengeMethod: string | null = null
     if (this.flowType === 'pkce') {
       const codeVerifier = generatePKCEVerifier()
-      await setItemAsync(this.storage, `${this.storageKey}-code-verifier`, codeVerifier)
+      await setItemAsync(
+        this.storage,
+        `${this.storageKey}-code-verifier`,
+        `${codeVerifier}/PASSWORD_RECOVERY`
```

Comment:
Using `/` as the separator here, becuase `/` is not used for generating code verifiers. 
https://github.com/supabase/gotrue-js/blob/master/src/lib/helpers.ts#L265